template/base/_gitignore

```diff
@@ -8,6 +8,9 @@
 # testing
 /coverage
 
+# database
+db.sqlite
```

Comment:
I think `db.sqlite` exists inside the prisma folder. So please change the path to `/prisma/db.sqlite`

Reply:
It works this way (see screenshots) but I think you're right that being specific about the path is better. Will change.
 
<img width="1552" alt="Screenshot 2022-07-16 at 12 43 09" src="https://user-images.githubusercontent.com/8353666/179351775-d626fbfd-20e2-4ea7-b90d-b9880772b3fc.png">
<img width="1552" alt="Screenshot 2022-07-16 at 12 42 55" src="https://user-images.githubusercontent.com/8353666/179351765-dbcd70de-42fc-429f-bb95-720809196e19.png">